packages/babel-generator/test/fixtures/importAttributesKeyword/attributes-with-to-default/options.json

```diff
@@ -1,4 +1,3 @@
 {
-  "BABEL_8_BREAKING": true,
```

Comment:
The `attributes-with-to-default` is merged with this case now that we output the standard syntax by default.